src/frontend-admin/src/components/reusables/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react'
+import styles from './Dropdown.module.css'
+import { DropdownList } from './DropdownList'
+//import title to put the button inside of the corner brackets
+
+interface IDropdownProps {
+    content: {name: string, item?: any, onClick?: Function}[]
+}
+
+export const Dropdown: React.FC<IDropdownProps> = (props) => {
+    const {content} = props
+    const [selected, setSelected] = useState(content[0])
+
+    return (
+        <div className={styles.dropdownMain}>
+        
+            <div className={styles.dropdownContainer}>
+                <DropdownList 
+                    triggerElement={({isOpen, toggle}) => (
+                        <button onClick={toggle} className={styles.dropdownButton}>
+                            <div className={styles.dropdownTitle}>{selected.name}
+                                <div className={styles.dropdownArrow}/>
```

Comment:
I like that this one is pure css - for anything more complicated I highly recommend using the react-icons library (which Alex is already using in another branch)